### PR TITLE
database cache from 2.x

### PR DESCRIPTION
**Pull request by alekseytupichenkov**
required https://github.com/liip/LiipFunctionalTestBundle/pull/398

### CODE REVIEW

File: src/Test/WebTestCase.php [hunk 1]
```diff
@@ -72,11 +72,6 @@
      */
     private $excludedDoctrineTables = [];
 
-    /**
-     * @var array
-     */
-    private static $cachedMetadatas = [];
```
Comment (alexislefebvre) [inline src/Test/WebTestCase.php]: I don't understand why this change is here, since it was removed in your previous PR: https://github.com/liip/LiipFunctionalTestBundle/pull/398/files#diff-3216cb8554ab63f3b7b8531ebcee5156

Maybe rebasing your PR on the `2.x` could fix this issue?

Comment (alekseytupichenkov) [reply src/Test/WebTestCase.php]: Hm.. I'll check it today

Comment (alekseytupichenkov) [reply src/Test/WebTestCase.php]: Done

File: src/Services/DatabaseBackup/MongodbDatabaseBackup.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Liip/FunctionalTestBundle
+ *
+ * (c) Lukas Kahwe Smith <smith@pooteeweet.org>
```
Comment (maximgubar) [inline src/Services/DatabaseBackup/MongodbDatabaseBackup.php]: does the code actually belongs to Lukas ?
